payments-core/src/main/java/com/stripe/android/networking/StripeRepository.kt

```diff
@@ -34,9 +35,10 @@ import java.util.Locale
 /**
  * An interface for data operations on Stripe API objects.
  */
-internal interface StripeRepository {
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) // originally made public for paymentsheet
```

Comment:
Would like to see an interface for this in the future.

Reply:
Interfaces can not be marked internal, by making this abstract we can control which of these methods are exposed.